.travis.yml

```diff
@@ -1,19 +1,20 @@
 language: php
 
 php:
-    - 5.4
-    - 5.5
     - 5.6
+    - 7.0
```

Comment:
PHP 7.0 is no longer maintained, I think we can remove it.

Before I approve, do you want to fix all the errors in this PR or fix only the memory-limit and retry the current PRs ?